OMCompiler/SimulationRuntime/c/simulation/solver/initialization/initialization.c

```diff
@@ -299,7 +301,7 @@ static int symbolic_initialization(DATA *data, threadData_t *threadData)
       else
         data->callback->functionInitialEquations(data, threadData);
 
-      infoStreamPrint(LOG_INIT, 0, "homotopy parameter lambda = %g done\n---------------------------", data->simulationInfo->lambda);
+      infoStreamPrint(LOG_INIT, 0, "homotopy parameter lambda = %g done\n---------------------------", lambda);
```

Comment:
@AnHeuermann 
This is from a previous commit of yours, i did not know why you did this so i didn't remove it. Could you explain or is it outdated and i should remove it?

Reply:
data->simulationInfo->lambda gets overwritten during asserts and this will result in a wrong log output.